ee/WFLY-14061_EAP7-1366_JavaEE-to-JakartaEE-refs.adoc

```diff
@@ -0,0 +1,58 @@
+= Replace JavaEE references with corresponding JakartaEE ones
+:author:            Alessio Soldano
+:email:             asoldano@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+As WildFly is JakartaEE 8 certified since version 18 (https://github.com/wildfly/certifications/blob/EE8/WildFly_18.0.0.Final/jakarta-full-platform.adoc) and JakartaEE 9 implementation is in progress, we want to properly position the project as a JakartaEE implementation more than a JavaEE one.
+
+== Issue Metadata
+
+=== Issue
+
+* {issue-base-url}/EAP7-1366 [EAP7-1366]
+* {issue-base-url}/WFLY-14061 [WFLY-14061]
+
+=== Related Issues
+
+* {issue-base-url}/EAP7-1193 [EAP7-1193]
+
+=== Dev Contacts
+
+* mailto:asoldano@redhat.com[Alessio Soldano]
+
+=== QE Contacts
+
+* Jan Blizňák
+
+=== Testing By
+
+[ ] Engineering
+
+[x] QE
+
+=== Affected Projects or Components
+
+* WildFly
```

Comment:
Is this really scoped to just WildFly and not all the (common) projects it integrates?

Reply:
So far it's scoped to WildFly, but time permitting (this has to go in 23) I think we could try applying changes to WildFly Core as well.

Reply:
This seems to be missing WildFly Core as we do have some EE integrations in there as well.